### ISSUE TITLE
Include a straightforward way to use the swtpm

### ISSUE THREAD

**Issue by lkatalin** [keylime/rust-keylime#259]
The way we are using it is undocumented and inconsistent with the documentation in `keylime/keylime`. It would be even better if there is a way to set this up the same way each time via an included optional script.

**Comment by mpeters** [comment]
The recent updates to the vagrant dev setup have this fixed. Not sure we need to do anything here exactly as it's very env dependent. Maybe include something in the docs about using the vagrant setup?

**Comment by lkatalin** [comment]
When I worked on this, it was very hard to onboard (myself and later other people) because the setup steps were undocumented. I think pointing to [these lines](https://github.com/keylime/keylime-vagrant-ansible-tpm-emulator/blob/master/roles/tpm_setup/files/tpm_emulator.service#L9-10) would be a good start but there are still some steps missing like doing the `modprobe`. The [gist](https://gist.github.com/lkatalin/74f6cf1fb0b3ba8123d47f02f9b17dcf) I wrote was pretty comprehensive but is probably outdated. My goal was to have well-documented setup steps for at least one or two environments. But as I don't have the time to add this right now, it's up to you all.
